cmd/ebrelayer/txs/relayToCosmos.go

```diff
@@ -49,20 +43,18 @@ func RelayToCosmos(factory tx.Factory, moniker, password string, claims []*types
 		}
 	}
 
-	sugaredLogger.Infow(
-		"relay sequenceNumber from builder.",
-		"nextSequenceNumber", nextSequenceNumber,
+	sugaredLogger.Infow("RelayToCosmos building, signing, and broadcasting", "messages", messages)
+	// TODO this WithGas isn't correct
+	// TODO we need to investigate retries
+	// TODO we need to investigate what happens when the transaction has already been completed
+	err := tx.BroadcastTx(
```

Comment:
Created a card for this in zenhub: https://app.zenhub.com/workspaces/sifchain-603ed608675da3001744d736/issues/sifchain/sifnode/1346 @banshee